### ISSUE TITLE
Extra Effect: Reverse Reverb

### ISSUE THREAD

**Issue by mikenrafter** [thedocruby/resounding#58]
**Is your feature request related to a problem? Please describe.**
Reverb sounds different when it's heading toward you, vs away.

**Describe the solution you'd like**
Reverse reverb! It adds a ghostly effect, when hearing things in semi-distant caves.

**Describe alternatives you've considered**
Irrelevant.

**Describe any compatibility requirements**
Not so much compatibility, whereas this might be performance-heavy, or require a specialized pipeline.

**Comment by thedocruby** [comment]
Could you elaborate on this? I'm not sure I understand what you are suggesting. Also, considering alternatives is **never** irrelevant, please fix, thx.

**Comment by ryleu** [comment]
Do you mean the Doppler effect?

**Comment by mikenrafter** [comment]
No, not the doppler effect, but after doing some more research, this effect doesn't actually occur in real sound physics...

It could be a potential extra, toggleable, effect just for mood/atmosphere, here's what I'm talking about: [reverse-reverb](https://blog.landr.com/reverse-reverb/).
